Index: src/com/leting/statistics/controller/StatisticsController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/leting/statistics/controller/StatisticsController.java	(date 1547535108000)
+++ src/com/leting/statistics/controller/StatisticsController.java	(revision )
@@ -31,7 +31,7 @@
 
         retMap = statisticsService.dataStatistics(request,response);
 
-        System.out.println(" 大盘鸡");
+        System.out.println(" 大盘鸡 ");
         System.out.println("吃没了");
         System.out.println("1");
         System.out.println("2");
